### ISSUE TITLE
aws-lc-sys --features ssl,bindgen failed to build

### ISSUE THREAD

**Issue by zh-jq** [aws/aws-lc-rs#348]
```
cargo build -p aws-lc-sys --features ssl,bindgen
```
failed to compile, the output:

```
error[E0425]: cannot find function, tuple struct or tuple variant `BIO_ctrl` in this scope
  --> aws-lc-sys/src/lib.rs:82:14
   |
82 |     unsafe { BIO_ctrl(b, BIO_CTRL_INFO, 0, pp.cast::<c_void>()) }
   |              ^^^^^^^^ not found in this scope

error[E0425]: cannot find function, tuple struct or tuple variant `CRYPTO_library_init` in this scope
  --> aws-lc-sys/src/lib.rs:86:14
   |
86 |     unsafe { CRYPTO_library_init() }
   |              ^^^^^^^^^^^^^^^^^^^ not found in this scope

For more information about this error, try `rustc --explain E0425`.
error: could not compile `aws-lc-sys` (lib) due to 2 previous errors
```

the function name in the bindgen.rs is
```
extern "C" {
    pub fn aws_lc_0_13_0_CRYPTO_library_init();
}
```
which is different than the pre-generated one:
```
extern "C" {
    #[link_name = "\u{1}aws_lc_0_13_0_CRYPTO_library_init"]
    pub fn CRYPTO_library_init();
}
```

**Comment by justsmth** [comment]
Thanks for the report. I yanked [v0.13.1](https://crates.io/crates/aws-lc-sys/0.13.1) so we can investigate.

**Comment by justsmth** [comment]
This was the result of a bad release of our sys-crates. The issue is addressed in [this PR](https://github.com/aws/aws-lc-rs/pull/349). Another release will be out soon.

**Comment by justsmth** [comment]
We've released [aws-ls-sys v0.13.2](https://crates.io/crates/aws-lc-sys/0.13.2).
